biz.aQute.bndlib/src/aQute/bnd/build/model/JavaSE_15.properties

```diff
@@ -463,10 +540,6 @@ jpms.modules = \
  jdk.nio.mapmode;\
   exports:List<String>='\
    jdk.nio.mapmode',\
- jdk.scripting.nashorn;\
```

Comment:
Can you also remove the modules with empty exports lists?

Reply:
will do!